public/locales/en/audit.json

```diff
@@ -169,8 +169,8 @@
                 "has-button": "has b<tag-underline>u</tag-underline>tton",
                 "button-waist-height": "button waist <tag-underline>h</tag-underline>eight",
                 "APS": "<tag-underline>A</tag-underline>PS",
-                "missing-crosswalk": "m<tag-underline>i</tag-underline>ssing crosswalk",
-                "no-bus-stop-access": "no bus stop <tag-underline>a</tag-underline>ccess",
+                "missing-crosswalk": "missing crosswalk (<tag-underline>a</tag-underline>)",
```

Comment:
you can underline the letter 'a' in 'missing crosswalk' instead of having it in parens at the end!